packages/search-input/docs/overview.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Search Input
+description: Search enables users to specify a word or a phrase to find relevant pieces of content without the use of navigation.
+group: Forms
+storybookPath: /story/forms-searchinput--basic
+---
+
+By default, the `SearchInput` component does not expand to fill the available space.
```

Comment:
Could we include some text here to explain that this is ideal for filtering items that have already loaded? and to suggest SearchBox if they require a different pattern.